src/components/FeedbackIndicator/FeedbackIndicator.js

```diff
@@ -17,8 +17,8 @@ const FeedbackIndicator = ({ status, ...props }) => {
         display: flex;
         align-items: center;
         justify-content: center;
-        width: 60px;
-        height: 60px;
+        width: 64px;
+        height: 64px;
```

Comment:
Let's `GU`ify these!
```suggestion
        width: ${8 * GU}px;
        height: ${8 * GU}px;
```